app/views/articles/show.html.erb

```diff
@@ -171,6 +169,9 @@
                     <% end %>
                   </em>
                 <% end %>
+                <% if should_show_updated_on?(@article) %>
+                  <em>Updated on <time datetime="<%= @article.edited_at&.utc&.iso8601 %>"><%= @article.edited_at&.strftime("%b %d, %Y") %></time></em>
```

Comment:
Maybe we can replace this with the `local_date` helper: https://github.com/forem/forem/blob/master/app/helpers/date_helper.rb